tests/Feature/LoginTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\Hash;
+
+class LoginTest extends TestCase
+{
+    
+    use DatabaseMigrations;
+    
+    protected function setUp():void
+    {
+        parent::setUp();
+        Artisan::call('migrate');
+        Artisan::call('db:seed');
+        Artisan::call('passport:install');
+    }
+    public function testLogin()
+    {
+        $user = User::create([
+            'email' => "asa@aa.aa",
+            'password' => Hash::make("123456789"),
+            'password_confirmation' => "123456789"
+        ]);        
+                
+        $response = $this->post('/api/login', [
+            'email' => "asa@aa.aa",
+            'password' => "123456789",
+            'password_confirmation' => "123456789"
+        ])->assertStatus(200)
+        ->assertJsonStructure(['token']);
+
+        $this->assertAuthenticatedAs($user); 
+        
+        $response = $this->post('/api/login', [
+            'password_confirmation' => "1234567"
+        ])->assertStatus(302);
```

Comment:
its better to test the 422 access error with correct email but wrong password - instead or aside of the validation test